modules/quadbin/snowflake/sql/QUADBIN_POLYFILL.sql

```diff
@@ -0,0 +1,53 @@
+----------------------------
+-- Copyright (C) 2022 CARTO
+----------------------------
+
+CREATE OR REPLACE FUNCTION QUADBIN_POLYFILL
+(geog GEOGRAPHY, resolution INT)
+RETURNS ARRAY
+AS $$
+    IFF(geog IS NULL OR resolution IS NULL,
+        NULL,
+        IFF(resolution < 0 OR resolution > 29,
```

Comment:
29 -> 26

Reply:
Also it looks like when we are out of resolutions we rather throw an error. And NULL is more for NULL inputs.